downstream/modules/eda/proc-eda-config-remote-sys-to-events.adoc

```diff
@@ -0,0 +1,32 @@
+[id="eda-config-remote-sys-to-events"] 
+
+= Configuring your remote system to send events
+
+After you have created your event stream, you must configure your remote system to send events to {EDAcontroller}. The method used for this configuration varies, depending on the vendor for the event stream credential type you select.
+
+.Prerequisites
+
+* The URL that was generated when you created your event stream
+* Secrets or passwords that you set up in your event stream credential
+
+.Procedure
+
+The following example demonstrates how to configure webhooks in a remote system like GitHub to send events to {EDAcontroller}. Each vendor will have unique methods for configuring your remote system to send events to {EDAcontroller}.
+
+. Log in to your GitHub repository.
+. Click *Your profile name → Your repositories*.
+
+[NOTE]
+====
+If you do not have a repository, click *New* to create a new one, select an owner, add a *Repository name*, and click *Create repository*.
+====
+
+. Navigate to *Settings* (tool bar).
+. In the *General* navigation pane, select *Webhooks*.
+. Click *Add webhook*.
+. In the *Payload URL* field, paste the URL you saved when you created your event stream.
+. Select *application/json* in the *Content type* list.
+. Enter your *Secret*.
+. Click *Add webhook*.
+
+After the webhook has been added, it attempts to send a test payload to ensure there is connectivity between the two systems (GitHub and {EDAcontroller}). If it can successfully send the data you will see a green check mark next to the *Webhook URL* with the message, *Last delivery was successful*.
```

Comment:
If it can successfully send the data, a check mark appears next to the * Webhook URL * with ...

Does a green check mark fit with accessibility advice?

Reply:
It probably needs a graphic. In the interest of time (as this topic has been heavily monitored), I decided I'll update after 2.5. If you can accept it for now, that will help us progress.